### PR TITLE
vscode-extensions: Updated op-vscode from 1.0.0 to 1.0.1

### PR DESCRIPTION
**Pull request by 2gn**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by 2gn** [comment]
I have a question: is it safe to add myself as a maintainer of this package? The extension itself was not built by myself, so I feel bad for the original author.

**Comment by superherointj** [comment]
> I have a question: is it safe to add myself as a maintainer of this package? The extension itself was not built by myself, so I feel bad for the original author.

Maintainer means the maintainer of the package at nixpkgs. Not upstream author.

**Comment by superherointj** [comment]
> I have a question: is it safe to add myself as a maintainer of this package? The extension itself was not built by myself, so I feel bad for the original author.

Maintainer means the maintainer of the package at nixpkgs. Not upstream author.

And you should also complete the meta information. And please, add yourself as maintainer. You'll be notified to review package when it updates.

**Comment by 2gn** [comment]
> > I have a question: is it safe to add myself as a maintainer of this package? The extension itself was not built by myself, so I feel bad for the original author.
> 
> Maintainer means the maintainer of the package at nixpkgs. Not upstream author.
> 
> And you should also complete the meta information. And please, add yourself as maintainer. You'll be notified to review package when it updates.

OK. Thank you!

**Comment by superherointj** [comment]
@2gn Meta information is still missing. Please complete it. Also include downloadPage (as marketplace download page).

**Comment by superherointj** [comment]
Also, there is an error, you need to add yourself as maintainer first. `2gn` does not seem valid.

https://gist.github.com/GrahamcOfBorg/d74ab5e64bf985938da1c50bb173eabb

**Comment by 2gn** [comment]
> Also, there is an error, `2gn` does not seem valid.

I forgot that I need to prepend an underscore to my username...

**Comment by superherointj** [comment]
It is still failing. I think you forgot to add "with lib;" at meta field.

Certify to build the package every time you change it. So you can pick-up all these errors early.

**Comment by 2gn** [comment]
> It is still failing. I think you forgot to add "with lib;" at meta field.
> 
> Certify to build the package every time you change it. So you can pick-up all these errors early.

After adding `with lib;` I ran the following command: 

```shell
nix-build -A pkgs.vscode-extensions.1Password.op-vscode
```

and it worked fine so this time it'll all be good. Sorry for bothering you.

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 182483` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>vscode-extensions._1Password.op-vscode</li>
  </ul>
</details>

File: pkgs/applications/editors/vscode/extensions/default.nix [hunk 1]
```diff
@@ -34,11 +34,16 @@ let
...
-          version = "1.0.0";
-          sha256 = "sha256-ZeKTP3WKjyuR/ryBdJRHXJT+l2gbY4QnWNTsN9+4nOA=";
+          version = "1.0.1";
+          sha256 = "sha256-0SsHf1zZgmrb7oIsRU6Xpa3AvR8bSfANz5ZlRogjiS0=";
         };
-        meta = {
-          license = lib.licenses.mit;
+        meta = with lib; {
+          changelog = "https://github.com/1Password/op-vscode/releases";
+          description = "1Password for VS Code";
```
Comment (superherointj) [inline pkgs/applications/editors/vscode/extensions/default.nix:42]:
```suggestion
          description = "A VS Code extension that integrates your development workflow with 1Password";
```

**Comment by superherointj** [comment]
Squash commits.

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 182483` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>vscode-extensions._1Password.op-vscode</li>
  </ul>
</details>

**Review by superherointj — APPROVED** [review]

**Pull request merged**